src/components/PackageLocationSelectionForm.vue

```diff
@@ -145,6 +200,9 @@ export default defineComponent({
         tableFullScreen(props){
             props.toggleFullscreen()
         },
+        nsvca (modul){
```

Comment:
same above, i'm think we already have same func in other place, can we reuse it?

Reply:
pls define `nsvca` func in one place and import this func for `Build`, `BuildFeedItem`, `PackageLocationSelectionForm`

Reply:
Done